src/main/java/module-info.java

```diff
@@ -24,6 +24,7 @@
 	exports org.junitpioneer.jupiter.params;
 	exports org.junitpioneer.jupiter.json;
 	exports org.junitpioneer.jupiter.converter;
+	exports org.junitpioneer.jupiter.displaynamegenerator;
```

Comment:
I created another package because this is a new purpose. I wanted the package name to be as clear as possible to the developer in order to easily guess the link with JUnit 5 DisplayNameGenerator.  Also, I thought there will be other DisplayName generators in the future. Furthermore,  testing this new feature requires to create other classes then grouping these test classes in the same package is a good idea.